typescript/core/src/McapIndexedReader.test.ts

```diff
@@ -142,15 +143,15 @@ describe("McapIndexedReader", () => {
 
     data.push(
       ...record(Opcode.FOOTER, [
-        ...uint64LE(BigInt(summaryStart)), // summary offset
-        ...uint64LE(0n), // summary start offset
+        ...uint64LE(BigInt(summaryStart)), // summary start
+        ...uint64LE(0n), // summary offset start
         ...uint32LE(crc32(new Uint8Array([42]))), // summary crc
       ]),
       ...MCAP_MAGIC,
     );
     const readable = makeReadable(new Uint8Array(data));
     await expect(McapIndexedReader.Initialize({ readable })).rejects.toThrow(
-      "Incorrect summary CRC 491514153 (expected 163128923)",
+      "Incorrect summary CRC 1656343536 (expected 163128923)",
```

Comment:
this changed because of the additional `DataEnd` record?

Reply:
yes, indirectly, because the summary CRC includes the CRC of `footer.summary_start + footer.summary_offset_start`